recipes/macfsevents/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "macfsevents" %}
+{% set version = "0.7" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "95d3cddaf8a42435bfbd50087785ee9e3ebb8325242cfd06a88ea21f85bcb56f" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.python.org/packages/f4/fb/59f72719e339f6209997414c6d0b7e1e1f96900dede3ec1cc24008471cc2/MacFSEvents-0.7.tar.gz
```

Comment:
Use the link as in the example recipe at https://github.com/conda-forge/staged-recipes/tree/master/recipes/example

Reply:
Yes, I started from the example + conda skeleton. This failed to build due to 301 responses from Pypi. (Other packages were building fine -- just this one had a problem). I hardcoded a direct link and that seemed to solve the problem.

I'm not sure why I was hitting redirects for this package. Best I can figure it might be related to [this issue](https://github.com/LLNL/spack/issues/1939)

Reply:
https://pypi.io/packages/source/M/MacFSEvents/MacFSEvents-0.7.tar.gz works for me.